spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/JSqlParserQueryEnhancer.java

```diff
@@ -319,7 +320,7 @@ private String detectAlias(String query) {
 			 * ValuesStatement has no alias
 			 * SetOperation can have multiple alias for each operation item
 			 */
-			if (!(selectStatement.getSelectBody()instanceof PlainSelect selectBody)) {
+			if (!(selectStatement.getSelectBody() instanceof PlainSelect selectBody)) {
 				return null;
```

Comment:
Whereas I sympathize with the desire to introduce this space WHICH SHOULD BE HERE, our current tools, due to `getSelectBody()` (the parentheses) will take that space back out the next time I'm inside this class and I reformat it. Hence, I'd rather not introduce that churn.